spec/webauthn/authenticator_data/attested_credential_data_spec.rb

```diff
@@ -20,14 +20,12 @@ def raw_attested_credential_data(options = {})
       attested_credential_data =
         WebAuthn::AuthenticatorData::AttestedCredentialData.new(raw_data)
 
-      expect(attested_credential_data.valid?).to be_falsy
-      expect(attested_credential_data.credential).to eq(nil)
+      expect { attested_credential_data.valid? }.to raise_error(COSE::UnknownKeyType)
+      expect { attested_credential_data.credential }.to raise_error(COSE::UnknownKeyType)
     end
 
-    it "returns false if one of public key coordinate is not long enough" do
-      raw_data = raw_attested_credential_data(
-        public_key: fake_cose_credential_key(y_coordinate: SecureRandom.random_bytes(31))
-      )
+    it "returns false if public key doesn't contain the alg parameter" do
```

Comment:
Does it add any value to add a test that check that `credential.public_key` is retrieving an intact version of the public_key, just as it was formed by the authenticator?

Reply:
Great call!